server/DataBase/models/user.js

```diff
@@ -36,7 +36,11 @@ let UserSchema = new mongoose.Schema({
         completedQuestions : {type : Number, default : 0},
         score : {type : Number, default : 0}
     }],
-    deleted : {type : Boolean, default : false},
+    deActivate : {type : Boolean, default : false},
+    deletedAt : {type : Date, default : null},
```

Comment:
as we are not storing deleted information so that will be deActivatedAt and deActivatedBy

Reply:
Done.